### ISSUE TITLE
Drupal Console Launcher > 1.1.0 breaks extend plugin

### ISSUE THREAD

**Issue by freelock** [hechoendrupal/drupal-console-launcher#144]
Hi,

I can no longer use the console-yaml extend plugin, with any launcher from 1.1.1 on... 1.1.0 is the last version that works.

I've been installing the Yaml plugin using these steps:

```
RUN mkdir ~/.console && cd ~/.console &&   composer create-project drupal/console-extend extend --no-interaction &&   cd extend && composer require drupal/console-yaml
```

... with DC 1.1.0 `drupal list yaml` gives a list of Yaml commands, and with DC 1.1.1 I get "[Symfony\Component\Console\Exception\CommandNotFoundException]  
  There are no commands defined in the "yaml" namespace."

**Comment by jmolivas** [comment]
@freelock  This fix https://github.com/hechoendrupal/drupal-console-launcher/issues/144 will be included in the next release 1.4.1

**Comment by jmolivas** [comment]
@freelock This should be fixed on 1.5.0 could you update to latest version and confirm.

**Comment by freelock** [comment]
Did not upgrade our CI container that needed this to 1.5.0, but confirming it works correctly with 1.6.0.
